nx-X11/config/cf/gnuLib.tmpl

```diff
@@ -0,0 +1,6 @@
+XCOMM
+XCOMM  GNU Hurd shared library template
+XCOMM
+
+#define SharedX11Reqs
+#define SharedXlibi18nReqs $(LDPRELIB) $(XONLYLIB)
```

Comment:
Hmm, I just came around that:

```
nx-X11/config/cf/README:        SharedXextReqs          required libs for libXext.so
nx-X11/config/cf/lnxLib.tmpl:#define SharedXextReqs       $(LDPRELIB) $(XONLYLIB)
```

Looks unused, but has nothing to do with this PR. We do have an Xext extension for the X server though, so I wonder if we messed up something before.